activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/contexts/generic.jsonld",
+        "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/activities/MedicalHistory-LORIS/medicalhistory_context.jsonld"
+    ],
+    "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Activity.jsonld",
+    "@id": "medicalhistory_sectionD",
+    "skos:prefLabel": "Section D of Medical History Instrument",
+    "skos:altLabel": "medicalhistory_sectionD",
+    "schema:description": "Section D of Medical History Instrument",
+    "schema:schemaVersion": "0.0.1",
+    "schema:version": "0.0.1",
+    "preamble": "D. Neurological",
+    "ui": {
+        "inputType": "section",
+        "order": [
+            "medicalhistory_21",
+            "medicalhistory_21a_incident",
+            "medicalhistory_21a_hospitalized",
+            "medicalhistory_21a_age",
+            "medicalhistory_21b_incident",
+            "medicalhistory_21b_hospitalized",
+            "medicalhistory_21b_age",
+            "medicalhistory_21c_incident",
+            "medicalhistory_21c_hospitalized",
+            "medicalhistory_21c_age",
+            "medicalhistory_21d"
+        ],
+        "shuffle": false,
+        "render": {
```

Comment:
to follow up from our conversation, i understand that we want activities with different renderings i.e. table vs not table to be separate activities. but then why is this way of doing it (with the render key) problematic? even though i prefer having the inputType: "table" implementation, i'm trying to understand why this way of implementing is not suitable @satra @akeshavan

Reply:
I think "render" is fine! If we know its a table, we'll (in theory) know to look at "render"